src/main/java/ru/itis/controllers/ProfileController.java

```diff
@@ -30,27 +28,28 @@ public ProfileController(WishListService wishListService, UserService userServic
     @GetMapping("/profile")
     @PreAuthorize("isAuthenticated()")
     @ApiOperation("View user profile page")
-    public Map<String, Object> getProfilePage(@RequestHeader(name = "Authorization") String token) {
+    public ResponseDto getProfilePage(@RequestHeader(name = "Authorization") String token) {
         Optional<User> userCandidate = userService.findUserByToken(token);
-        return ResponseUserDto.from(userCandidate.orElse(User.getDefaultUser()));
+        return userCandidate.map(user -> new ResponseDto(HttpStatus.OK, userService.userToMap(user))).orElseGet(() ->
+                new ResponseDto(HttpStatus.OK, "Invalid token"));
     }
 
     @CrossOrigin
     @PostMapping("/profile")
     @PreAuthorize("isAuthenticated()")
     @ApiOperation("Create new wish list")
-    public WishList createNewWL(@RequestParam String title, @RequestHeader("Authorization") String token) {
-        return wishListService.addNewWishList(title, token);
+    public ResponseDto createNewWL(@RequestParam String title, @RequestHeader("Authorization") String token) {
+        return new ResponseDto(HttpStatus.OK, wishListService.addNewWishList(title, token));
     }
 
     @CrossOrigin
     @DeleteMapping("/profile")
     @PreAuthorize("isAuthenticated()")
     @ApiOperation("Delete a wish list")
-    public ResponseEntity<?> deleteWishList(@RequestParam String title, @RequestHeader("Authorization") String token) {
+    public ResponseDto deleteWishList(@RequestParam String title, @RequestHeader("Authorization") String token) {
         if (!wishListService.removeByTitle(title, token)) {
-            return new ResponseEntity<>(HttpStatus.NOT_FOUND);
+            return new ResponseDto(HttpStatus.OK, "Invalid wish list title");
```

Comment:
Is it OK? No, it's not. Respond with 404. Maybe `ReponseEntity.notFound` or  I don't know.

Reply:
fixed